iterators.tex

```diff
@@ -276,7 +276,7 @@
   template <class I>
   concept bool Readable@\newtxt{() \{}\oldtxt{ =}@
     @\newtxt{return}@ Semiregular<I>@\newtxt{()}@ &&
-      requires (I i) {
+      requires (@\newtxt{const }@I i) {
```

Comment:
We use "identical semantics" in the description of "implicitly declared expression variants" and "semantically identical to" in the requirements for Boolean. Without definition. I'm reasonably certain that we mean something like "equivalent as observable by equality-preserving expressions" which is a bit looser than the standard's notion of "observable side effects."